CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# MDN Web Docs contribution guide
+
+Thanks for taking the time to contribute to [MDN Web Docs](https://developer.mozilla.org)' translated content! :tada:
+
+This document covers project setup steps along with a set of guidelines for contributing to MDN Web Docs translated content.
+Everyone participating in this project is expected to follow our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Getting started
+
+Before contributing to translated content, we strongly recommend learning about [how to contribute to the original (English) content][Contributing guide] (often referred to as "upstream content"). The upstream contributing guide explains the guidelines and conventions of MDN pages, as well as setup instructions for more significant changes.
+
+We also strongly recommend reading the [Translation guidelines][].
+
+In addition to the requirements described in the upstream contributing guide, there are additional prerequesites you must have when contributing to this repository:
+
+- **Moderate knowledge of the English language:** You must have a good enough understanding of the English language to be able to decipher the meaning of a page while translating. (For simple typo fixes, this is not required.)
+- **Fluency in the downstream locale:** You must be able to fluently speak the language you are contributing to.
+
+### Simple changes
+
+If you want to make a small change like fixing a typo, the GitHub UI is the easiest way to get started.
+If you've found a typo on the Simplified Chinese [JavaScript landing page](/zh-CN/docs/Web/JavaScript), for example, you can propose a fix as follows:
+
+1. Sign in to [GitHub](https://github.com/)
+2. Navigate to [https://github.com/mdn/translated-content](https://github.com/mdn/translated-content)
+3. Find the source file `files/zh-cn/web/javascript/index.md`
+4. Click the edit (pencil) button
+
+From there, the GitHub UI will walk you through the rest by creating a [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and a branch to commit your changes to.
```

Comment:
Should we remove the `locale` for all the urls pointing to GItHub docs? Those documents may be localized, readers could see documentation in their preferred language.

```suggestion
From there, the GitHub UI will walk you through the rest by creating a [fork](https://docs.github.com/get-started/quickstart/fork-a-repo) and a branch to commit your changes to.
```